src/build/env.py

```diff
@@ -89,5 +89,12 @@ def install(self, requirements):  # type: (Iterable[str]) -> None
 
         subprocess.check_call([sys.executable, '-m', 'ensurepip'], cwd=self._path)
 
-        cmd = [sys.executable, '-m', 'pip', 'install', '--ignore-installed', '--prefix', self._path] + list(requirements)
-        subprocess.check_call(cmd)
+        with tempfile.NamedTemporaryFile('w+', prefix='build-reqs-', suffix='.txt') as req_file:
+            req_file.write(os.linesep.join(requirements))
+            req_file.flush()
+            cmd = [
+                sys.executable, '-m', 'pip',
```

Comment:
This implies pip as install dependency... but it does not show up here https://github.com/FFY00/python-build/blob/master/setup.cfg#L24

Reply:
No? This is run inside the isolated environment, and we use ensurepip to get pip there.

Reply:
Can you comment that here, or link to where we use ensurepip as such?

Reply:
It's literally just up there. It is included in the preview.

```python
subprocess.check_call([sys.executable, '-m', 'ensurepip'], cwd=self._path)
```